csharp/ql/src/Telemetry/ExternalApiSpecific.qll

```diff
@@ -0,0 +1,80 @@
+/** Provides classes and predicates related to handling APIs from external libraries. */
+
+import csharp
+import semmle.code.csharp.dataflow.DataFlow
+import semmle.code.csharp.dataflow.ExternalFlow
+import semmle.code.csharp.dataflow.FlowSummary
+import semmle.code.csharp.dataflow.TaintTracking
+import semmle.code.csharp.security.dataflow.flowsources.Remote
+private import semmle.code.csharp.dataflow.internal.DataFlowPrivate
+private import semmle.code.csharp.dataflow.internal.TaintTrackingPrivate
+
+/**
+ * An external API from either the C# Standard Library or a 3rd party library.
+ */
+class ExternalApiSpecific extends Callable {
```

Comment:
Extends `DataFlowCallable` (i.e., only unbounds)?

Reply:
Yes, you are right. This should also be aligned with the TargetApi implementation, which also now extends DatFlowCallable. Also; We are only interested in unbound version as these are the ones we make summaries for.